README.md

```diff
@@ -46,24 +46,10 @@ Links to presentations, discussions and interviews are available in
 To get started and contribute to the project, visit the [ Technical setup](./TECHNICAL-SETUP.md) page and the
 [Contributing](./CONTRIBUTING.md) page.
 
-A Cosmos Cash testnet is available at these coordinates:
 
-- **Chain ID**: `cosmoscash-testnet`
-- **Token Denom**: `cash`
-- **Genesis**: [cosmos-cash.app.beta.starport.cloud/genesis](https://cosmos-cash.app.beta.starport.cloud/genesis?)
-- **RPC URL**:  `https://rpc.cosmos-cash.app.beta.starport.cloud:443`
+#### Testnet
 
-A Cosmos Cash DID resolver endpoint that is compatible with
-the [universal resolver](https://github.com/decentralized-identity/universal-resolver) driver specifications is
-available at:
-
-```url
-https://resolver-driver.cosmos-cash.app.beta.starport.cloud/identifier/
-```
-
-A universal resolver frontend is also available for testing and verification purposes at:
-
-- https://uniresolver.cosmos-cash.app.beta.starport.cloud
+To join or interact using the testnet, refer to the [testnet coordinates page](https://docs.cosmos-cash.app.beta.starport.cloud/Networks/testnet/)
```

Comment:
```suggestion
To join or interact using the testnet, see [testnet coordinates](https://docs.cosmos-cash.app.beta.starport.cloud/Networks/testnet/)
```

Reply:
hi @noandrea let's change the "refer to the" to "see" and just use the descriptive link text

can we use a relative link here?

Reply:
no, we have to have the fully qualified URL